tests/sqlparser_common.rs

```diff
@@ -7762,7 +7762,7 @@ fn parse_trim() {
         Box::new(MySqlDialect {}),
         //Box::new(BigQueryDialect {}),
         Box::new(SQLiteDialect {}),
-        Box::new(DuckDbDialect {}),
+        //Box::new(DuckDbDialect {}),
```

Comment:
```suggestion
```
I think it should be fine to remove the dialect from the list

Reply:
I commented it out to match the existing code style where BigQuery and Snowflake were commented out as well.